README.md

```diff
@@ -9,62 +9,32 @@ GET /v1/cost/ping
 GET /v1/cost/version
 GET /v1/cost/metrics
 
-GET /v1/cost/{account}/spaces/{spaceid}[?start=2019-10-01&end=2019-10-30]
+GET /v1/cost/{account}/spaces/{spaceid}[?start=2019-10-01&end=2019-10-30][&groupBy=SERVICE]
```

Comment:
So is this going to replace the old (expensive) way of getting cost estimates per resource type?

Reply:
unclear, but I did use it to track down costs for data transfer a while back, so I thought we might expose it